app/models/user.rb

```diff
@@ -1,16 +1,21 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] I can see that the confirmable is there but it is not effected, when I signed up, it should ensure I click the link on the console to confirm my email before taking me to the app. The previous reviewer did not get to see this because he could not even signup at all and this is a core requirement.